pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/metadata-step/metadata-step.component.html

```diff
@@ -100,56 +100,48 @@ <h4 i18n="metadata cluster settings" aria-level="2">
             <ng-template #hasLabels>
                 <div class="clr-row label-container" *ngFor="let label of labels | keyvalue">
                     <clr-input-container class="label-input">
-                        <input clrInput [value]="label.key" [disabled]="true" aria-label="label key"/>
+                        <input clrInput formControlName="{{label.key}}" placeholder="key" aria-label="label key"/>
                         <!-- Add a empty helper to avoid accessibility aria-describeby issue -->
                         <clr-control-helper></clr-control-helper>
+                        <clr-control-error i18n="cluster label valid msg" *clrIfError="validatorEnum.VALID_CLUSTER_NAME" class="err-label">
+                            {{ clusterTypeDescriptorTitleCase }} Cluster Label Keys must start and end with an alphanumeric character, and can contain only
+                            letters, numbers, hyphens, underscores, and dots.
+                        </clr-control-error>
+                        <clr-control-error i18n="cluster label valid msg" *clrIfError="validatorEnum.WHITESPACE" class="err-label">
+                            {{ clusterTypeDescriptorTitleCase }} Cluster Label Keys must not include whitespace on ends.
+                        </clr-control-error>
+                        <clr-control-error i18n="cluster label valid msg" *clrIfError="validatorEnum.MAX_LEN" class="err-label">
+                            {{ clusterTypeDescriptorTitleCase }} Cluster Label Keys max length is 63 characters.
+                        </clr-control-error>
+                        <clr-control-error i18n="cluster label valid msg" *clrIfError="validatorEnum.LABEL_UNIQUE" class="err-label">
+                            A Label with the same key already exists.
+                        </clr-control-error> 
                     </clr-input-container>
                     <span class="label-input">:</span>
                     <clr-input-container class="label-input">
-                        <input clrInput [value]="label.value" [disabled]="true" aria-label="label value"/>
+                        <input clrInput formControlName="{{label.value}}" placeholder="value" aria-label="label value"/>
                         <!-- Add a empty helper to avoid accessibility aria-describeby issue -->
                         <clr-control-helper></clr-control-helper>
+                        <clr-control-error i18n="cluster label valid msg" *clrIfError="validatorEnum.VALID_CLUSTER_NAME" class="err-label">
+                            {{ clusterTypeDescriptorTitleCase }} Cluster Label Values must start and end with an alphanumeric character, and can contain only
```

Comment:
Nitpick: please change Cluster Label Values from being all capitalized to Cluster label values, so only 'Cluster' is caps. This goes for all three error messages. Thanks!

Reply:
fixed